src/test/java/com/amazon/opendistroforelasticsearch/jobscheduler/JobSchedulerPluginTests.java

```diff
@@ -16,11 +16,11 @@
 package com.amazon.opendistroforelasticsearch.jobscheduler;
 
 import org.junit.Assert;
-import org.junit.Test;
+// import org.junit.Test;
```

Comment:
Two things - What was reason for removing? And if you comment something out, just completely remove it, we can always refer to git history if we need to get something back, commented out code is just dead code that isn't used so don't be shy to get rid of it.

Reply:
IDE shows me `@Test` annotation is not needed, a method named like `testFooBar` is enough. 
Guess comment but not remove is a bad habit of me, will correct that, thanks for point out.